src/Ombi/Controllers/V1/TokenController.cs

```diff
@@ -80,6 +84,7 @@ public async Task<IActionResult> GetToken([FromBody] UserAuthModel model)
                 var url = await _plexOAuthManager.GetOAuthUrl(model.PlexTvPin.code, websiteAddress);
                 if (url == null)
                 {
+                    _log.LogWarning(string.Format("Failed login attempt by IP: {0}", GetRequestIP()));
```

Comment:
This is not really a failure due to the user here

Reply:
You're right, thanks!